client/state/jetpack/credentials/actions.js

```diff
@@ -19,11 +19,13 @@ export const getCredentials = ( siteId ) => ( {
 	siteId,
 } );
 
-export const updateCredentials = ( siteId, credentials, stream = false ) => ( {
+
+export const updateCredentials = ( siteId, credentials, stream = false, shouldUseNotices = true ) => ( {
```

Comment:
To resolve a tiny linter warning:

```suggestion
export const updateCredentials = (
	siteId,
	credentials,
	stream = false,
	shouldUseNotices = true
) => ( {
```